extensions/amp-next-page/0.1/amp-next-page.js

```diff
@@ -266,17 +68,15 @@ export class AmpNextPage extends AMP.BaseElement {
         `${TAG} config should ` +
             'be inside a <script> tag with type="application/json"');
     const configJson = tryParseJson(scriptElement.textContent, error => {
```

Comment:
Manually creating the error is unnecessary. You should use `user().error()` like `amp-state` uses:

https://github.com/ampproject/amphtml/blob/c574297b6f5be642cd7cfd05299be11b4cea6345/extensions/amp-bind/0.1/amp-state.js#L123-L126

Reply:
Done.